docs/howto/generating-liquidity-pool-rewards.md

```diff
@@ -0,0 +1,61 @@
+---
+description: Generating liquidity pool rewards
+---
+
+# How to generate liquidity pool rewards
+
+When using [uniswap](https://uniswap.org), you can stake your SAND in liquidity pool to earn rewards tokens.
+
+## Requirements
+
+If you haven't done it, follow the [getting started](../intro/index.md) tutorial first.
+
+Here the inputs you need to get a snapshot of everyone who staked SAND up until a date of your choice:
+
+- the date to consider for the rewards (it will determine the block number)
+
+## The Graph service
+
+The script located at `scripts/gathering/generateLPRewards.ts` is using the service [The Graph](https://thegraph.com) to query the blockchain. A [subgraph API](https://api.thegraph.com/subgraphs/name/pixowl/staking) has been deployed and used by the script to determine the rewards with a graphql query.
```

Comment:
maybe leave the link https://api.thegraph.com/subgraphs/name/pixowl/staking/graphql  as text instead of a hyperlink